projects/netgrif-components-core/src/lib/utility/base64.spec.ts

```diff
@@ -0,0 +1,22 @@
+import {decodeBase64, encodeBase64} from "./base64";
+
+
+describe('Base64Utils', () => {
+
+    it('should encode/decode non-latin characters', (done) => {
+        const nonLatin = 'ššščččôôôžžťťť';
```

Comment:
why does this test exist, if the other one tests a super-set of these characters?

Reply:
usually the test falls on these characters....